### PR TITLE
Add Aya keyboard

### PR DESCRIPTION
**Pull request by DanNixon**
## Description

Adds target for [Aya](https://github.com/DanNixon/aya) keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/aya/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+![Aya](https://raw.githubusercontent.com/DanNixon/aya/main/docs/both_sides.jpg)
+
+A column staggered, symmetrical split keyboard.
+
+* Keyboard Maintainer: [Dan Nixon](https://github.com/dannixon)
+* Hardware Supported: Aya PCB, Pro Micro
+* Hardware Availability: CAD files [here](https://github.com/DanNixon/aya)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/aya/readme.md:11]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/aya/info.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "Aya",
+    "url": "https://github.com/DanNixon/aya",
+    "maintainer": "dannixon",
+    "width": 19,
+    "height": 5,
```
Comment (fauxpark) [inline keyboards/aya/info.json:6]:
```suggestion
```

File: keyboards/aya/rules.mk [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+COMMAND_ENABLE = yes
+NKRO_ENABLE = no
+BACKLIGHT_ENABLE = no
+AUDIO_ENABLE = no
+UNICODE_ENABLE = no
+BLUETOOTH_ENABLE = no
+RGBLIGHT_ENABLE = no
+SLEEP_LED_ENABLE = no
+
+SPLIT_KEYBOARD = yes
```
Comment (fauxpark) [inline keyboards/aya/rules.mk:17]:
```suggestion
# MCU name
MCU = atmega32u4

# Bootloader selection
BOOTLOADER = caterina

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
SPLIT_KEYBOARD = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**